hack/update-vendor-licenses.sh

```diff
@@ -161,7 +161,7 @@ echo "= Kubernetes licensed under: ="
 echo
 cat "${LICENSE_ROOT}/LICENSE"
 echo
-echo "= LICENSE $(cat "${LICENSE_ROOT}/LICENSE" | md5sum | awk '{print $1}')"
+echo "= LICENSE $(< "${LICENSE_ROOT}/LICENSE" md5sum | awk '{print $1}')"
```

Comment:
change reason:
```
In hack/update-vendor-licenses.sh line 164:
echo "= LICENSE $(cat "${LICENSE_ROOT}/LICENSE" | md5sum | awk '{print $1}')"
                      ^-- SC2002: Useless cat. Consider 'cmd < file | ..' or 'cmd file | ..' instead.
```

Reply:
Can you reformat this so the command comes first? e.g. `md5sum < file`. This improves readability.